kafka-questdb-connector-samples/stocks/docker-compose.yml

```diff
@@ -27,14 +26,14 @@ services:
     links:
       - questdb:questdb
     environment:
-        - GF_SECURITY_ADMIN_PASSWORD=quest
-        - GF_SECURITY_ADMIN_USER=admin
-        - GF_USERS_ALLOW_SIGN_UP=false
-        - GF_USERS_ALLOW_ORG_CREATE=false
-        - GF_AUTH_ANONYMOUS_ENABLED=true
-        - GF_AUTH_ANONYMOUS_ORG_ROLE=Admin
-        - GF_AUTH_ANONYMOUS_ORG_NAME=Main Org.
-        - GF_AUTH_ANONYMOUS_ORG_ID=1
+      - GF_SECURITY_ADMIN_PASSWORD=quest
```

Comment:
whitespace changes in a yaml file makes me anxious :)
but I assume you tried it, didn't you?

Reply:
Sorry that was accidental!